coalib/bears/requirements/PythonRequirement.py

```diff
@@ -0,0 +1,45 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+
+
+class PythonRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    more requirements from ``pip``, without writing the name of the class
```

Comment:
hm I guess you're right, the method ``multiple`` helps you do that, not the class itself.